_projects/2024/100541120/100541120.Rmd

```diff
@@ -0,0 +1,997 @@
+Paises_Bajos <- readPNG(flagon::flags("nl"))
+Irlanda_norte <- readPNG("C:/Users/jorge/Desktop/CSS/Data-Visualization/Cocaine in Europe - Datavis/Images/Northern Ireland.png")
+Uruguay <- readPNG(flagon::flags("uy"))
+Argentina <- readPNG(flagon::flags("ar"))
+Irlanda <- readPNG(flagon::flags("ie"))
+Canada <- readPNG(flagon::flags("ca"))
+Polonia <- readPNG(flagon::flags("pl"))
+Chile <- readPNG(flagon::flags("cl"))
+CostaRica <- readPNG(flagon::flags("cr"))
+
+#Add flags to the bar chart using `annotation_raster`
+flag_plot <- barchart_coc1 +
+  annotation_raster(CostaRica, xmin = 0.6, xmax = 1.3, ymin = -0.62, ymax = -0.34) +  
+  annotation_raster(Chile, xmin = 1.6, xmax = 2.3, ymin = -0.62, ymax = -0.34) +  
+  annotation_raster(Polonia, xmin = 2.6, xmax = 3.3, ymin = -0.62, ymax = -0.34) +  
+  annotation_raster(Canada, xmin = 3.6, xmax = 4.3, ymin = -0.62, ymax = -0.34) +  
+  annotation_raster(Argentina, xmin = 5.6, xmax = 6.3, ymin = -0.62, ymax = -0.34) +  
+  annotation_raster(Irlanda, xmin = 4.6, xmax = 5.3, ymin = -0.62, ymax = -0.34) + 
+  annotation_raster(Irlanda_norte, xmin = 7.6, xmax = 8.3, ymin = -0.62, ymax = -0.34) +  
+  annotation_raster(Uruguay, xmin = 6.6, xmax = 7.3, ymin = -0.62, ymax = -0.34) +  
+  annotation_raster(Paises_Bajos, xmin = 8.6, xmax = 9.3, ymin = -0.62, ymax = -0.34) + 
+  annotation_raster(España, xmin = 9.6, xmax = 10.3, ymin = -0.62, ymax = -0.34) + 
+  annotation_raster(Inglaterra, xmin = 10.6, xmax = 11.3, ymin = -0.62, ymax = -0.34) +  
+  annotation_raster(Escocia, xmin = 11.6, xmax = 12.3, ymin = -0.62, ymax = -0.34) +  
+  annotation_raster(Estados_Unidos, xmin = 12.6, xmax = 13.3, ymin = -0.62, ymax = -0.34) +  
+  annotation_raster(Albania, xmin = 13.6, xmax = 14.3, ymin = -0.62, ymax = -0.34) + 
+  annotation_raster(Australia, xmin = 14.6, xmax = 15.3, ymin = -0.62, ymax = -0.34)  
+
+
+#Now, we can display the final plot with flags
+flag_plot
+
+```
+
+
+As we can see, one problem I encountered was with the quality of the flags in the graph. Despite using the annotation_raster() function to add them, the quality of the flags appeared quite poor.
+For the shades that has the authentic graph, I tried to introduce grey rectangles on the back of the flags but it wasn't a good option. Also I tried to edit the images of the packages but then when I tried to introduce those flags in the graph they dissappeared. 
+
+
+The next step is to introduce the logos. These original figures were difficult to find, so I decided to made a screenshot of the original plot and introduce the pictures with patchwork package. 
+
+```{r, fig.width= 8.5, fig.height= 6}
+
+
+#logos
+library(patchwork)  
+library(grid)  
+library(png)  
+
+# Read the PNG images for the logos
+logo1 <- readPNG("C:/Users/jorge/Desktop/CSS/Data-Visualization/Cocaine in Europe - Datavis/eom.png")
+logo2 <- readPNG("C:/Users/jorge/Desktop/CSS/Data-Visualization/Cocaine in Europe - Datavis/cuch.png")  
```

Comment:
Please include these images (in general, any external files) in the PR close to this Rmd, then call them just by their name, without any path (note that these paths are in YOUR computer).